lib/src/androidTest/java/com/couchbase/lite/internal/core/C4QueryBaseTest.java

```diff
@@ -116,20 +118,25 @@ protected C4Query compile(String whereExpr, String sortExpr, boolean addOffsetLi
     protected List<List<List<Long>>> runFTS(Map<String, Object> params) throws LiteCoreException {
         List<List<List<Long>>> matches = new ArrayList<>();
         C4QueryOptions opts = new C4QueryOptions();
-        C4QueryEnumerator e = query.run(opts, encodeParameters(params));
+        AllocSlice encodedParams = encodeParameters(params);
+        C4QueryEnumerator e = query.run(opts,encodedParams);
```

Comment:
space needed after the comma, in the parameters section. 
C4QueryEnumerator e = query.run(opts, encodedParams);

Reply:
Thanks.